styles/main.css

```diff
@@ -365,6 +365,16 @@ footer ul li a:focus {
     margin-left: auto;
     margin-right: auto;
     display: block;
+    filter: url("data:image/svg+xml;utf8,<svg version='1.1' xmlns='http://www.w3.org/2000/svg' height='0'><filter id='greyscale'><feColorMatrix type='matrix' values='0.3333 0.3333 0.3333 0 0 0.3333 0.3333 0.3333 0 0 0.3333 0.3333 0.3333 0 0 0 0 0 1 0' /></filter></svg>#greyscale");
+    -webkit-filter: grayscale(100%);
+    filter: grayscale(100%);
+    filter: gray;
+}
+
+.column:hover img {
```

Comment:
Why don't to use class on img tag? 
Advantages:
* no cascades
* more evident code
* one class for one appereance